mod_reforged/hooks/skills/actives/drums_of_war_skill.nut

```diff
@@ -0,0 +1,20 @@
+::Reforged.HooksMod.hook("scripts/skills/actives/drums_of_war_skill", function(q) {
+	q.create = @(__original) function()
+	{
+		__original();
+		// Vanilla is missing a description for this skill
+		this.m.Description = "Strike a vigorous rythm making your allies feel less tired and more eager to fight.";
+	}
+
+	q.getTooltip = @(__original) function()
+	{
+		local ret = __original();
```

Comment:
This will not display AP and Fatigue cost. Use `this.skill.getDefaultUtilityTooltip` instead.